src/main/java/com/ubb/jobs/repo/impl/RequestRepo.java

```diff
@@ -26,12 +26,17 @@ public RequestDto addRequest(RequestDto request) {
     }
 
     public List<RequestDto> getRequestByClient(int id){
-        List<Request> requests =requestRepo.findRequestByUserTo(id);
+        List<Request> requests =requestRepo.findRequestByUserFrom(id);
         return requestMapper.toDtos(requests);
     }
 
     public List<RequestDto> getRequestByProvider(int id){
-        List<Request> requests =requestRepo.findRequestByUserFrom(id);
+        List<Request> requests =requestRepo.findRequestByUserTo(id);
         return requestMapper.toDtos(requests);

```

Comment:
get by provider si client is inversate.

Client -> are nevoie de get by  client -> userFrom
Provider -> are nevoie de get by provider -> userTo

Reply:
posibil sa ai dreptate, pe mine deja ma baga in ceata care care ii :)